setup.py

```diff
@@ -371,12 +371,13 @@ def get_data_files():
     },
     'entry_points': {
         'console_scripts': [
+            'jupyter-voila = voila.app:main',
             'voila = voila.app:main'
         ]
     },
     'install_requires': [
         'async_generator',
-        'jupyter_server>=0.1.0,<0.2.0',
+        'jupyter_server>=0.2.0.dev0,<0.3.0',
```

Comment:
Is `.dev0` still needed now that `0.2.0` has been released?

Reply:
Yes. I'd set the version to `>=0.2.1, <0.3.0`.

Reply:
:+1: